### PR TITLE
Delete the hierarchy

### PR DESCRIPTION
**Pull request by Sameerthe-Big-O**

### CODE REVIEW

**Comment by Sameerthe-Big-O** [comment]
just noticed i haven't use if else conditions when the i'm getting the response back from the services

File: config/env.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 const environment = {
   port: process.env.PORT || 2022,
   nodeEnv: process.env.NODE_ENV || "development",
-  mongodbUri: process.env.DB_URI || "mongodb://127.0.0.1:27017/boilerplate",
+  mongodbUri: process.env.DB_URI || "mongodb://127.0.0.1:27017/HRMS",
```
Comment (AbubakarZorrain) [inline config/env.js:4]: Use Live MongoDb.

Comment (AbubakarZorrain) [reply config/env.js:4]: Why env  is in env.js?

File: dsd [hunk 2]
```diff
@@ -0,0 +1,10 @@
+  create-hierarchy[m
```
Comment (AbubakarZorrain) [inline dsd:1]: What is this file doing

File: services/hierarchy.js [hunk 3]
```diff
@@ -9,15 +9,18 @@ const hierarchySerive = {
...
-      if (savedData) {
-        return { message: "success", data: savedData };
-      }
-    } catch (error) {
-      return { message: "error", data: error.message };
+  get: async (id)=>{
+    try{
+        const data = await hierarchyModel.findById(id);
+        if(data){
+        return { message: "success", data };
```
Comment (AbubakarZorrain) [inline services/hierarchy.js:16]: Do not send message from service layer. just send data. handle message in controller.

File: services/hierarchy.js [hunk 4]
```diff
@@ -31,26 +34,30 @@ const hierarchySerive = {
...
-      const savedData = await hierarchyModel.findByIdAndUpdate(id, body);
-      if (savedData) {
-        return { message: "success", data: savedData };
-      }
-    } catch (error) {
-      return { message: "error", data: error.message };
+  update:async (id,body)=>{
+    try{
+        const savedData=await hierarchyModel.findByIdAndUpdate(id,body);
+        //*first checking if the hierarchy even exist on this id
```
Comment (AbubakarZorrain) [inline services/hierarchy.js:40]: Remove comments...

**Review by AbubakarZorrain — CHANGES_REQUESTED** [review]

**Review by Sameerthe-Big-O — COMMENTED** [review]
review changes

**Review by AbubakarZorrain — CHANGES_REQUESTED** [review]
Resolve Conflicts

**Review by AbubakarZorrain — CHANGES_REQUESTED** [review]
Resolve Conflicts...

**Pull request closed without merging**